staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -96,6 +97,7 @@ func newGenericWebhook(scheme *runtime.Scheme, codecFactory serializer.CodecFact
 	clientConfig.ContentConfig.NegotiatedSerializer = serializer.NegotiatedSerializerWrapper(runtime.SerializerInfo{Serializer: codec})
 
 	clientConfig.Dial = customDial
+	clientConfig.Wrap(x509metrics.MissingSANRoundTripperWrapper(x509MissingSANCounter))
```

Comment:
does this cover admission webhooks and CRD conversion webhooks?

Reply:
It covers the webhooks that use this generic code, which is admission, imagepolicy, webhook authenticator and authorizer.

The CRD conversion webhook seems to be using it's custom logic, I'll look into adding it there, too.

Reply:
actually, I noticed the admission webhooks othen than imagepolicy work differently, I'll need to address that, too

Reply:
Looks like there're fortunately only two different ways used when constructing a webhook, possibly because some webhooks are constructed from config, others from live API objects. I guess it might be possible to unify this but that's outside of what this PR tries to do.

Anyway, I wired the roundtripper wrapping in the `HookClient` method of the `ClientManager` struct which seems to be present for all the "dynamic" webhooks, but not in the ones using `NewGenericWebhook` constructor so double wrapping should not happen.